src/programs/refine3d/refine3d.cpp

```diff
@@ -971,6 +971,9 @@ bool Refine3DApp::DoCalculation()
 	input_3d_local.CopyAllButVolume(&input_3d);
 	input_3d_local.density_map = input_3d.density_map;
 
+	bool global_search_local = global_search;
+	bool local_refinement_local = local_refinement;
```

Comment:
These variable names are pretty confusing, because "local" can either refer to a local refinement, or to thread-local (I guess). How about `global_searh_this_thread` and `local_refinement_this_thread`?

Reply:
I agree, but I was going with consistency - all the other variables specific to the thread are _local.  Would you refactor them all?

Reply:
For this file, I think I would because of the other meaning of `local`. Just think it would make things easier for future readers of the code. But if you're opposed, I'm not that concerned so I'll approve anyway in case you just want to get this over with :)